internal/kv/mem/mem_kv.go

```diff
@@ -42,6 +42,8 @@ type memoryKVItem struct {
 	key, value string
```

Comment:
questions: why to we change 
memoryKVItem to 	key string, value interface{} so value can be any type and we don't need to do type conversion each time?

Reply:
we denitely need generics mechenism in Go 1.18 since I hate the code like this @congqixia @sunby

Reply:
Before introducing generics, performance impact shall be evaluated first. I think an interface like 
```go
type Value struct {
      String() string
      ByteSlice() []string
}
```
will do the trick.